agent/hcp/telemetry/otel_sink.go

```diff
@@ -20,8 +20,15 @@ import (
 // DefaultExportInterval is a default time interval between export of aggregated metrics.
 const DefaultExportInterval = 10 * time.Second
 
+// Disabled should be implemented to turn on/off metrics processing
+type Disabled interface {
```

Comment:
Went back and forth between "`Disabler`, `Disablable` which would allow is to have just an `Enabled()` method...

This one makes the method `IsEnabled()` but the name is better?